src/thor/costmatrix.cc

```diff
@@ -1164,7 +1164,7 @@ void CostMatrix::SetTargets(baldr::GraphReader& graphreader,
                              !opp_dir_edge->not_thru(), !(costing_->IsClosed(directededge, tile)),
                              static_cast<bool>(flow_sources & kDefaultFlowMask),
                              InternalTurn::kNoTurn, kInvalidRestriction,
-                             static_cast<uint8_t>(costing_->Allowed(opp_dir_edge, opp_tile)),
+                             static_cast<uint8_t>(costing_->Allowed(directededge, tile)),
```

Comment:
which field of the edgelabel is this, hard to review this part

Reply:
this is `path_id`, the one that's used to store whether the edge is allowed in the initial edge labels